alf/environments/suite_robotics.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Suite for loading OpenAI Robotics environments.
+
+**NOTE**: Mujoco requires separated installation.
+
+(gym >= 0.10, and mujoco>=1.50)
+
+Follow the instructions at:
+
+https://github.com/openai/mujoco-py
+
+"""
+
+try:
+    import mujoco_py
+except ImportError:
+    mujoco_py = None
+
+import functools
+import gym
+from gym.wrappers import FlattenDictWrapper
+
+import gin
+from alf.environments import suite_gym, torch_wrappers, process_environment
+from alf.environments.utils import UnwrappedEnvChecker
+
+_unwrapped_env_checker_ = UnwrappedEnvChecker()
+
+
+def is_available():
+    return mujoco_py is not None
+
+
+class SparseReward(gym.Wrapper):
+    """Convert the original :math:`-1/0` rewards to :math:`0/1`.
+    """
+
+    def __init__(self, env):
```

Comment:
maybe use threshold = 0.5 as an argument, < thresh gives 0, above gives 1?
This makes the wrapper less task dependent, and also avoids potential problems with float comparison.

Reply:
This wrapper is only meant for openai Robotics environment. it shouldn't be used for other envs